### PR TITLE
Fix policy trigger for action execution cancellation

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

File: st2actions/st2actions/notifier/notifier.py [hunk 1]
```diff
@@ -112,12 +111,12 @@ def _post_notify_triggers(self, liveaction=None, execution_id=None):
...
                 notify_subsection=notify.on_complete,
                 default_message_suffix='completed.')
-        if liveaction.status == LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
             self._post_notify_subsection_triggers(
                 liveaction=liveaction, execution_id=execution_id,
                 notify_subsection=notify.on_success,
                 default_message_suffix='succeeded.')
-        if liveaction.status == LIVEACTION_STATUS_FAILED and notify.on_failure:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_FAILED and notify.on_failure:
```
Comment (manasdk) [inline st2actions/st2actions/notifier/notifier.py]: TIMED_OUT is also a failure I think we need to handle that here as well.

Comment (Kami) [inline st2actions/st2actions/notifier/notifier.py]: Yeah, I'm just working on another PR - it also updates action chain to use this new status and fixes and updates some more places to correctly check for all failed and completed states.

Comment (Kami) [inline st2actions/st2actions/notifier/notifier.py]: But yeah, do feel free to point out if you spot another place which you think I missed and we should update it.

Comment (Kami) [inline st2actions/st2actions/notifier/notifier.py]: In my PR I also introduced `action_constants.COMPLETED_STATES` so we don't copy and paste and duplicate those constants all over...

Comment (Kami) [inline st2actions/st2actions/notifier/notifier.py]: Also, lol, I first thought you commented on my PR, but that's @m4dcoder's PR.

In any case, my branch / PR is WIP and should be ready on Monday (https://github.com/StackStorm/st2/compare/retry_policy_improvements).

**Comment by Kami** [comment]
@m4dcoder Good catch - I'm working on some more related changes (https://github.com/StackStorm/st2/compare/retry_policy_improvements).

Feel free to merge this if you want and I can reconcile my changes later.

**Pull request merged**